### PR TITLE
feat: remove support links

### PR DESCRIPTION
**Pull request by MrMartinR**
Fix for https://github.com/parse-community/parse-dashboard/issues/2184

### CODE REVIEW

**Comment by MrMartinR** [comment]
Hi Guys (@mtrezza, @moumouls, @dblythy, @cbaker6, @davimacedo, @dplewis) I think I am going to give up on this.
I removed the footer with the links, the mpn install showed some critical and high security stuff, I spent 1 hour trying to make the repo run, also because a bug in the passport dependency 0.6.0 (I had to install the 0.5.0)
I have no idea how to do tests and is something that I really do not want to learn right now (or maybe never 🤣🤔)
This PR supposed to "fix" this issue https://github.com/parse-community/parse-dashboard/issues/2184
If someone has the system set up and know how to pass the workflow requirements, and want to fix this, it will takes 2 minutes to delete the footer.

**Comment by mtrezza** [comment]
@MrMartinR I don't think the passport dep has anything to do with this PR, simply removing the links like what you've done in this PR should be enough. I suggest you check your local node version, and if you cannot get dashboard to run locally, you could ask for help in chat.

Also:
- Please always keep and use the template in issues and PRs.
- If you need support to make a PR, please state clearly where you are stuck so others can understand and help.
- If you need to bring attention to your PR, you could use the chat or tag the repo specific group, here `@parse-community/dashboard` rather than mentioning random users.
- As you can see in the CI window, the commit message / PR title test fails; if you click on the link next to it, you can read a description for how a PR title needs to be formatted. I changed it already.

**Comment by MrMartinR** [comment]
Hi @parse-community/dashboard !, all the checks passed, can someone review this?

File: src/components/Sidebar/FooterMenu.react.js [hunk 1]
```diff
@@ -1,69 +0,0 @@
...
-
-    let content = null;
-    if (this.state.show) {
-      content = (
-        <Popover
-          fixed={true}
-          position={this.state.position}
-          onExternalClick={() => this.setState({ show: false })}>
-          <div className={styles.popup}>
-            <a href={`${mountPath}logout`}>Log Out <span className={styles.emoji}>👋</span></a>
```
Comment (mtrezza) [inline src/components/Sidebar/FooterMenu.react.js:53]: You removed the whole footer, but that also removes the "Log Out" menu.

image


I suggest you go through the code parts that you remove line by line and see whether there is anything that should not be removed.

**Review by mtrezza — CHANGES_REQUESTED** [review]

File: src/components/Sidebar/FooterMenu.react.js (hunk 1 above)
Comment (MrMartinR) [reply src/components/Sidebar/FooterMenu.react.js:53]: Oh, that was a good one 🫢
Screen Shot 2022-07-03 at 15 00 06

**Review by mtrezza — COMMENTED** [review]
If I'm not mistaken, I don't think there is a confirmation dialog before logging out. Maybe it's best for UX to keep the logout menu as a submenu, so it requires 2 clicks. That will also make it easier to add more menu options in the future.

And I'd remove that hand emoji. It's a style that isn't used anywhere else in the dashboard.

**Comment by MrMartinR** [comment]
Yeah, I am not very happy with the emoji neither.. 
so, what do you propose? 3 dots button with the Log Out sub-menu?

**Comment by mtrezza** [comment]
Yes, maybe keep the 3 dots and only leave the Logout without the hand emoji? There are some icons in the dashboard repo where you may find a log out icon, but you don't really need to dig into that, not even sure if there is one.

**Comment by MrMartinR** [comment]
There are no logout/exit icon, probably in the future I will come back to this and I will create a logout icon, I am not 100% happy with the css I had to tweak because is just 1 element in the popover, probably in the future when I am more familiar with the code I will improve that.

**Comment by mtrezza** [comment]
Could you test out https://github.com/parse-community/parse-dashboard/pull/2203 and see if it works? I've made a few changes to make the button larger when used with a touch UI and changes the menu text.

I actually opened the PR because this PR did not display the pop-up correctly when I was testing it out, so I thought you needed some help, but it tuns out that your button works fine. So you can either incorporate the changes in this PR and I'll close the other one, or you can close this one and I'll merge the other one, whatever you prefer. If you want to use this PR, please look for unnecessary changes like added empty lines or spaces, these should be removed.

**Comment by mtrezza** [comment]
Superseded by https://github.com/parse-community/parse-dashboard/pull/2203

**Pull request closed without merging**